src/Compilers/CSharp/Portable/Binder/InContainerBinder.cs

```diff
@@ -101,6 +102,33 @@ internal override ImportChain ImportChain
             }
         }
 
+        internal override HashSet<string> QuickTypeIdentifierAttributeCheckSet
+        {
+            get
+            {
+                if (_lazyQuickTypeIdentifierAttributeCheckSet == null)
+                {
+                    HashSet<string> previousSet = this.Next.QuickTypeIdentifierAttributeCheckSet;
+                    HashSet<string> newSet = null;
+
+                    if ((object)_container == null || _container.Kind == SymbolKind.Namespace)
```

Comment:
:bulb: If you're going to have the cast to `(object)`, you may as well use `_container is null`.